core/data.go

```diff
@@ -75,19 +75,23 @@ type BlobRequestHeader struct {
 	SecurityParams []*SecurityParam `json:"security_params"`
 }
 
-func (sp *SecurityParam) Validate() error {
-	if sp.ConfirmationThreshold < sp.AdversaryThreshold || sp.ConfirmationThreshold-sp.AdversaryThreshold < 10 {
-		return errors.New("invalid request: quorum threshold must be >= 10 + adversary threshold")
+func ValidateSecurityParam(confirmationThreshold, adversaryThreshold uint32) error {
```

Comment:
It seems like maybe we should be checking that these parameters will be valid `uint8`s?

Reply:
The range is checked below: they cannot even be greater than 100, so that's even stricter than unit8

Reply:
lol, good point